packages/contracts/contracts/ChugSplashRegistry.sol

```diff
@@ -131,27 +130,21 @@ contract ChugSplashRegistry is Ownable, Initializable {
         address managerImpl = versions[_version.major][_version.minor][_version.patch];
         require(managerImplementations[managerImpl], "ChugSplashRegistry: invalid manager version");
 
-        address payable managerProxyAddress = getChugSplashManagerProxyAddress(
-            msg.sender,
-            _organizationID
-        );
-
-        projects[msg.sender][_organizationID] = managerProxyAddress;
-        managerProxies[managerProxyAddress] = true;
-
         bytes32 salt = keccak256(abi.encode(msg.sender, _organizationID));
 
-        // Deploy the ChugSplashManager proxy and set the implementation to the requested version
         ChugSplashManagerProxy managerProxy = new ChugSplashManagerProxy{ salt: salt }(
             this,
             address(this)
         );
 
         require(
-            address(managerProxy) == managerProxyAddress,
-            "ChugSplashRegistry: manager proxy not deployed correctly"
+            address(managerProxy) != address(0),
```

Comment:
I believe this `!= address(0)` check is sufficient to ensure that the create2 call was successful. [OpenZeppelin does the same thing.](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Create2.sol#L37)